### DISCUSSION TITLE
动态过滤貌似会把视频简介里的关键字也过滤掉，有误伤

### DISCUSSION THREAD

**Discussion by Zacccdp** [the1812/Bilibili-Evolved#4253] [功能建议 / Ideas]
关键词里添加”淘宝”了以后，会造成在简介有“淘宝”二字的视频也会被屏蔽掉
![Screenshot 2023-06-13 115911](https://github.com/the1812/Bilibili-Evolved/assets/74779279/0ba74367-ffb5-40da-b490-835056a9562a)
关键词加了“淘宝”视频就被屏蔽了
![Screenshot 2023-06-13 115945](https://github.com/the1812/Bilibili-Evolved/assets/74779279/10fa71f1-f309-406e-841c-f877be498f74)
可不可以让关键词忽略简介的内容呢？毕竟一般来说简介的内容不太会影响动态

**Comment by the1812** [comment]
过滤的时候区分不了这个, 动态卡片能解析出的文字都会算进去
